src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1353,4 +1355,61 @@ private AdsDataProvider() {
                 }
         };
     }
+
+  @DataProvider
+  public static Object[][] adsVelesTracking() {
+    final String TURN_ON_VELES = "?InstantGlobals.wgAdDriverVelesBidderCountries=[XX]";
+
+    return new Object[][]{
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles" + TURN_ON_VELES),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "20.00")
+                .put(AdsContent.INCONTENT_PLAYER, "20.00")
+                .build()
+        },
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Incontent" + TURN_ON_VELES),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "NOT_INVOLVED")
+                .put(AdsContent.INCONTENT_PLAYER, "20.00")
+                .build()
+        },
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Leaderboard" + TURN_ON_VELES),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "20.00")
+                .put(AdsContent.INCONTENT_PLAYER, "NOT_INVOLVED")
+                .build()
+        }
+    };
+  }
+
+  @DataProvider
+  public static Object[][] adsVelesErrorTracking() {
+    final String TURN_ON_VELES = "?InstantGlobals.wgAdDriverVelesBidderCountries=[XX]";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)